_src/edx-pl/sass/components/_forms.scss

```diff
@@ -0,0 +1,410 @@
+// ------------------------------
+// edX Pattern Library: Components - Forms
+
+// About: Contains base styling for forms
+
+// #GLOBAL
+// #INPUT TEXT
+// #INPUT RADIO/CHECKBOX
+// #SELECT
+// #TEXTAREA
+// #SLIDERS
+// #PROGRESS
+
+// ----------------------------
+// #GLOBAL
+// ----------------------------
+.form {
+    display: inline-block;
```

Comment:
Remove: This seems like a layout specific rule, not an across the board all forms type of rule. If you need it for the pattern library rendering, you can move it into that stylesheet.